library/src/scala/quoted/Quotes.scala

```diff
@@ -2571,6 +2571,9 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
         @experimental
         def substituteTypes(from: List[Symbol], to: List[TypeRepr]): TypeRepr
 
+        /** The type parameters */
```

Comment:
Should mention that these are the arguments of a type application and also what happens if it is not a type application.